src/app/domain/schema-management/components/manual-verification-wizard.component.html

```
@@ -0,0 +1,122 @@
+
+    <!-- Step 3: Audit Log -->
+    @if (currentStep() === 2) {
+      <div class="space-y-4">
+        <h3 class="text-lg font-medium text-main">3. Audit Trail Documentation</h3>
+        <p class="text-sm text-gray-600 dark:text-slate-300">
+          Record this unique SHA-256 Audit Hash in your study logs. It mathematically proves the integrity of the verified randomization dataset.
+        </p>
+        
+        <div class="bg-slate-900 dark:bg-slate-950 p-4 rounded-lg flex items-center justify-between">
+          <code class="text-sm text-green-400 break-all font-mono">{{ auditHash() }}</code>
+          <button (click)="copyHash()" class="ml-4 flex-shrink-0 inline-flex items-center px-3 py-1.5 border border-transparent text-xs font-medium rounded-lg text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 transition-colors">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)